lms/templates/dashboard.html

```diff
@@ -58,6 +58,11 @@
       isEdx: false
     });
   </%static:require_module>
+  <%static:require_module module_name="js/learner_dashboard/entitlement_unenrollment_factory" class_name="EntitlementUnenrollmentFactory">
+    EntitlementUnenrollmentFactory({
+      dashboardUrl: "${reverse('dashboard') | n, js_escaped_string}"
```

Comment:
This is where you can pass through the selectors

Reply:
I thought about doing it that way, but `EntitlementUnenrollView` is only meant to be used with `lms/templates/dashbaord/_dashboard_entitlement_unenrollment_modal.html`. Therefore, I felt it better to just include all the expected selectors in the Backbone view since they should never change. This is also DRYer as they only need to be defined once. If I define them here, I'd need to define them again in `themes/edx.org/lms/templates/dashboard.html`.